src/components/report/VariableControls.js

```diff
@@ -53,14 +50,24 @@ export default {
 			}
 			this.activeInstance = activeInstance;
 		},
-		getVariableStyles: function(axes) {
+		getVariableStyles: function() {
 			let styles = "";
 			let glue = "";
-			for (const axis of Object.values(axes)) {
+			for (const axis of Object.values(this.axes)) {
 				styles += `${glue} "${axis.id}" ${axis.current}`;
 				glue = ",";
 			}
 			return `font-variation-settings:${styles};`;
+		},
+		getInstanceStyles: function(instance) {
+			const axes = this.instances[instance];
+			let styles = "";
+			let glue = "";
+			for (const axis in axes) {
```

Comment:
This is becoming a bit of theme (😛), but how about:

```js
const variationSettings = axes.map(axis => `"${axis}" ${axes[axis]}`).join(",");
```

May the FP force be with you ;-)

Reply:
Thanks! Slightly different code as instance axes are objects, but this looks cleaner, thanks!